deployment/docker-headless/run-env-linux.sh

```diff
@@ -1,38 +0,0 @@
-#!/bin/bash
```

Comment:
We don't need this script anymore?

Reply:
Nope. I think we won't need it since we are using docker-compose to override settings.